src/modules/course/schema/enrolledCourse.schema.ts

```diff
@@ -0,0 +1,68 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import * as mongoose from 'mongoose';
+import { CourseSchema } from './course.schema';
```

Comment:
'CourseSchema' is defined but never used

Reply:
Will remove that import.. was not needed